torch/csrc/jit/passes/onnx/pattern_conversion/common.cpp

```diff
@@ -0,0 +1,45 @@
+#include <torch/csrc/jit/passes/onnx/pattern_conversion/common.h>
+
+namespace torch {
+namespace jit {
+
+bool IndexingPatternFinder::IsSameSource(const Node* n, const Node* m) {
+  const auto& source_n = n->sourceRange().source();
+  const auto& source_m = m->sourceRange().source();
+  return (
+      (source_n->text() == source_m->text()) &&
```

Comment:
Is this based on code line number? How does it work for the example below (values %8 and %11) ?

Reply:
yes same source code line. for the example below, %8 originates from `1:3` and %11 originates from `0` in `x[1:3, 0]` in the same line of source code.